recipes/ldcpy/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "ldcpy" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 820e9902787d4b93c111d9fa2d7a3b7af56c1f5b8084551530e0183df5eef072
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - cartopy =0.18
+    - netcdf4
+    - pip
+    - xarray
+    - zfpy
+    - xrft
+    - cmocean
+    - numpy
+    - scipy
+    - matplotlib
+    - dask
+    - intake-esm
+
+test:
+  imports:
+    - ldcpy
```

Can we try to ad ``pip check`` here please?
```suggestion
    - ldcpy
  requires:
    - pip
  commands:
    - pip check
```

---

Added, thanks!